pkg/api/graph/graphview/veneering_test.go

```diff
@@ -386,13 +390,13 @@ func TestGraph(t *testing.T) {
 			for _, image := range deployment.Images {
 				t.Logf("%s  image %s", indent, image.Image.ImageSpec())
 				if image.Build != nil {
-					if image.Build.LastSuccessfulBuild != nil {
-						t.Logf("%s    built at %s", indent, image.Build.LastSuccessfulBuild.CreationTimestamp)
-					} else if image.Build.LastUnsuccessfulBuild != nil {
-						t.Logf("%s    build %s at %s", indent, image.Build.LastUnsuccessfulBuild.Status, image.Build.LastSuccessfulBuild.CreationTimestamp)
+					if image.LastSuccessfulBuild != nil {
```

Comment:
> seems weird for a single image to have edges to multiple builds

`Images` aren't images, they are `ImagePipelines` which are closer to `ImageStreams`